packages/components/tests/integration/components/hds/table/index-test.js

```diff
@@ -10,8 +10,14 @@ module('Integration | Component | hds/table/index', function (hooks) {
     await render(hbs`<Hds::Table />`);
     assert.dom(this.element).exists();
   });
+
```

Comment:
[blocker] I would probably add more tests for the main `Table` component:
- that renders correctly the children for the different use cases
- that the table sorting functionality works as expected
- that renders the live-region caption, with the expected text, and that gets updated when sorting change

Reply:
I added tests for the second two items, can you say more about the first one?

Reply:
I'm going to add more tests after I finish a couple of tiny features. So I think I will consider this resolved for now.

Reply:
sorry, I don't see the tests for the second two items, is it me? 🤔

also, regarding the first item, what I mean is to test the different possible contents:
- table with no model
- table with model
- table with model + sortable

and for each one of them, check that the expected cells (tr/th and tr/td) are rendered correctly

Reply:
These have been moved to the integration test. Please see the PR description for the command to invoke the tests for this branch and a screenshot of the list of tests itself.